pkg/build/graph/analysis/bc.go

```diff
@@ -98,3 +102,79 @@ func FindCircularBuilds(g osgraph.Graph) []osgraph.Marker {
 
 	return markers
 }
+
+func FindPendingTags(g osgraph.Graph) []osgraph.Marker {
+	markers := []osgraph.Marker{}
+
+ist:
+	for _, uncastIstNode := range g.NodesByKind(imagegraph.ImageStreamTagNodeKind) {
+		istNode := uncastIstNode.(*imagegraph.ImageStreamTagNode)
+		if bcNode, points := buildPointsToTag(g, uncastIstNode); points && !istNode.Found() {
+
+			buildNode := latestBuild(g, bcNode)
+
+			if buildNode == nil {
+				// A build config points to the non existent tag but no current build exists.
+				markers = append(markers, osgraph.Marker{
+					Node:         bcNode,
+					RelatedNodes: []graph.Node{uncastIstNode},
+
+					Severity:   osgraph.WarningSeverity,
+					Key:        TagNotAvailableWarning,
+					Message:    fmt.Sprintf("%s needs to be imported or created by a build.", istNode.ResourceString()),
+					Suggestion: osgraph.Suggestion(fmt.Sprintf("oc start-build %s", bcNode.(*buildgraph.BuildConfigNode).ResourceString())),
+				})
+				continue ist
```

Comment:
So my previous comment doesn't get lost: unless I am missing something, there's no need to use the `ist` tag here, just using `continue` will have the correct effect.